tests/helm_inflator.sh

```diff
@@ -56,7 +56,7 @@ assert_contains_string out.yaml "name: extra-args-redis-master"
 
 helm_testcase "kpt_helm_inflator_imperative_pipeline"
 kpt fn source example-configs |
-  kpt fn run --mount type=bind,src="$(pwd)/${CHARTS_SRC}",dst=/source --image gcr.io/kpt-functions/helm-inflator:"${TAG}" --as-current-user -- local-chart-path=/source/zookeeper name=my-zookeeper | 
+  kpt fn run --mount type=bind,src="$(pwd)/${CHARTS_SRC}",dst=/source,rw=true --image gcr.io/kpt-functions/helm-inflator:"${TAG}" --as-current-user --network -- local-chart-path=/source/zookeeper name=my-zookeeper | 
```

Comment:
The tests here need to be refactored per @mengqiy doc.

Currently, these tests use test data from the SDK repo and charts from https://github.com/bitnami/charts.git which is very brittle. Test data and dependencies should either live in the repo or if remote (e.g. Helm chart), needs to use a particular version. Otherwise tests will break randomly like this.

Reply:
Agree, all e2e tests are needed to be re-written after tests framework finalized.